### PR TITLE
Add autoSender plugin

### PR DESCRIPTION
**Pull request by NcxyZero**
new file:   src/userplugins/autoSender/index.ts

It's a simple plugin that automates message sending. Provide the channel id, desired delay and the message in plugin's settings and enable it

### CODE REVIEW

**Comment by programminglaboratorys** [comment]
there is a reason src/userplugins is in gitignore so ppl don't accidently upload their userplugins. and automate? isn't that the main definition of a botting (aka selfbotting)

**Comment by NcxyZero** [comment]
> there is a reason src/userplugins is in gitignore so ppl don't accidently upload their userplugins. and automate? isn't that the main definition of a botting (aka selfbotting)

That makes sense but I don't know if this is a problem when as stated in README.md in disclaimer section that even vencord is against tos of discord

File: src/plugins/autoSender/index.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+export default definePlugin({
+    name: "Auto Sender",
+    description: "Send messages automaticaly to specifed channel",
+    authors: [
+        {
+            id: 322076647589412864n,
+            name: "Noxy",
+        },
+    ],
```
Comment (nin0-dev) [inline src/plugins/autoSender/index.ts:79]: Put yourself in the constants.ts file

**Review by nin0-dev — CHANGES_REQUESTED** [review]
For the sake of it I reviewed but don't expect this to be merged, that will likely get you terminated. 
That's just a glorified selfbot

**Review by nin0-dev — COMMENTED** [review]
Since I forgot some other comments here they are, take that as general advice if you ever want to make other plugins

**Comment by k26pl** [comment]
> That makes sense but I don't know if this is a problem when as stated in README.md in disclaimer section that even vencord is against tos of discord
Discord doesnt care about mods, but selfbots are bannable

**Comment by NcxyZero** [comment]
> > That makes sense but I don't know if this is a problem when as stated in README.md in disclaimer section that even vencord is against tos of discord
> 
> Discord doesnt care about mods, but selfbots are bannable

In that case I think I'm closing this

**Pull request closed without merging**